.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "vendor/assets/stylesheets/pygments-css"]
 	path = vendor/assets/stylesheets/pygments-css
 	url = https://github.com/richleland/pygments-css.git
+[submodule "vendor/assets/javascripts/recorderjs"]
+	path = client/vendor/recorderjs
+	url = https://github.com/mattdiamond/Recorderjs.git
+	ignore = dirty
```

Comment:
Should we remove this flag, since I think it's not really required?

Reply:
After we install dependencies for ```recorderjs```, we will also change some files in the ```recorderjs``` folder. If we remove this flag, git will remind us files changed in the ```recorderjs``` folder, which is annoying.

Reply:
Ic, ok then we can keep this here 👍, although my preference would be to move to some proper gem / npm package which I know doesn't exist yet.